From eaa4d30d5abd753bd08322e4b7e5ebd8a52dd75a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Oct 2013 12:47:34 +0200
Subject: [PATCH] dracut-bash-completion.sh: add --prelink --noprelink

---
 dracut-bash-completion.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-bash-completion.sh b/dracut-bash-completion.sh
index f822058..d796c35 100644
--- a/dracut-bash-completion.sh
+++ b/dracut-bash-completion.sh
@@ -34,7 +34,7 @@ _dracut() {
                               --local --hostonly --no-hostonly --fstab --help --bzip2 --lzma
                               --xz --no-compress --gzip --list-modules --show-modules --keep
                               --printsize --regenerate-all --noimageifnotneeded --early-microcode
-                              --no-early-microcode --print-cmdline'
+                              --no-early-microcode --print-cmdline --prelink --noprelink'
 
                        [ARG]='-a -m -o -d -I -k -c -L --kver --add --force-add --add-drivers
                               --omit-drivers --modules --omit --drivers --filesystems --install
