From 352a49a21d5093955e45c0ccb84c258d4b375aad Mon Sep 17 00:00:00 2001
From: Alexander Tsoy <alexander@tsoy.me>
Date: Fri, 25 Oct 2013 17:06:44 +0200
Subject: [PATCH] usrmount/module-setup.sh: fixed typo

---
 modules.d/98usrmount/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/98usrmount/module-setup.sh b/modules.d/98usrmount/module-setup.sh
index a668069..1dde937 100755
--- a/modules.d/98usrmount/module-setup.sh
+++ b/modules.d/98usrmount/module-setup.sh
@@ -6,7 +6,7 @@ check() {
     local _init
     [[ $mount_needs ]] && return 1
     _init=$(readlink -f /sbin/init)
-    [[ "$init" == "${init##/usr}" ]] && return 255
+    [[ "$_init" == "${_init##/usr}" ]] && return 255
     return 0
 }
 
