From 20ff2d37cc14f72a97325b12ebb7437efbaf0a8e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 4 Nov 2013 17:29:15 +0100
Subject: [PATCH] fips: also install /etc/system-fips in the initramfs

---
 modules.d/01fips/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/01fips/module-setup.sh b/modules.d/01fips/module-setup.sh
index 1ab4999a..35cfdc5a 100755
--- a/modules.d/01fips/module-setup.sh
+++ b/modules.d/01fips/module-setup.sh
@@ -41,5 +41,6 @@ install() {
         libssl.so 'hmaccalc/sha512hmac.hmac' libssl.so.10
 
     inst_multiple -o prelink
+    inst_simple /etc/system-fips
 }
 
