From ea9b961e8cffc30f131d62d21bcbd18b0482b7f6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 8 Jan 2014 15:37:18 +0100
Subject: [PATCH] dracut.sh: add missing piece for option --add-device

---
 dracut.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dracut.sh b/dracut.sh
index bb7e91e..9a6521e 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -307,6 +307,7 @@ TEMP=$(unset POSIXLY_CORRECT; getopt \
     --long add-fstab: \
     --long mount: \
     --long device: \
+    --long add-device: \
     --long nofscks: \
     --long ro-mnt \
     --long kmoddir: \
