From 9d47a90b12dd5aa34488be2d5e6ff082c6713b8d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 17 Jan 2014 12:16:41 +0100
Subject: [PATCH] systemd/rootfs-generator.sh: exit 0

---
 modules.d/98systemd/rootfs-generator.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/98systemd/rootfs-generator.sh b/modules.d/98systemd/rootfs-generator.sh
index 3770c6b1..555027f3 100755
--- a/modules.d/98systemd/rootfs-generator.sh
+++ b/modules.d/98systemd/rootfs-generator.sh
@@ -28,3 +28,5 @@ case "$root" in
 esac
 
 [ "${root%%:*}" = "block" ] && wait_for_dev -n "${root#block:}"
+
+exit 0
