From dccf47fd55e99700c930a23327bfb6647501c38a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 28 Feb 2014 13:26:13 +0100
Subject: [PATCH] test/TEST-50-MULTINIC/client-init.sh: skip MAC marker files

---
 test/TEST-50-MULTINIC/client-init.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/TEST-50-MULTINIC/client-init.sh b/test/TEST-50-MULTINIC/client-init.sh
index f13c3790..51f8b642 100755
--- a/test/TEST-50-MULTINIC/client-init.sh
+++ b/test/TEST-50-MULTINIC/client-init.sh
@@ -9,6 +9,7 @@ export PS1='initramfs-test:\w\$ '
 stty sane
 echo "made it to the rootfs! Powering down."
 for i in /run/initramfs/net.*.did-setup; do
+	strstr "$i" ":*:*:*:*:" && continue
 	i=${i%.did-setup}
 	IFACES+="${i##*/net.} "
 done
