From e2e32cd02febaf26b9cd0b17de9f02256dd72cb2 Mon Sep 17 00:00:00 2001
From: WANG Chao <chaowang@redhat.com>
Date: Mon, 3 Mar 2014 15:22:00 +0800
Subject: [PATCH] Add -[-no]-hostonly-cmdline option handling for getopt

commit ab9457e introduce such new options but it missed to add these
options to getopt arguments.

Signed-off-by: WANG Chao <chaowang@redhat.com>
---
 dracut.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dracut.sh b/dracut.sh
index 2d5b7dc..78d7fad 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -347,6 +347,8 @@ TEMP=$(unset POSIXLY_CORRECT; getopt \
     --long host-only \
     --long no-hostonly \
     --long no-host-only \
+    --long hostonly-cmdline \
+    --long no-hostonly-cmdline \
     --long persistent-policy: \
     --long fstab \
     --long help \
