From af1154fdce0e6fe0e8cb1216cc5e2f420eb03975 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 12 Mar 2014 15:39:08 +0100
Subject: [PATCH] systemd/dracut-shutdown.service: make failure non-fatal

---
 modules.d/98systemd/dracut-shutdown.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/98systemd/dracut-shutdown.service b/modules.d/98systemd/dracut-shutdown.service
index e733300c..3b8b6da6 100644
--- a/modules.d/98systemd/dracut-shutdown.service
+++ b/modules.d/98systemd/dracut-shutdown.service
@@ -15,6 +15,6 @@ ConditionPathExists=/run/initramfs/.need_shutdown
 ConditionPathExists=!/run/initramfs/bin/sh
 
 [Service]
-ExecStart=/usr/lib/dracut/dracut-initramfs-restore
+ExecStart=-/usr/lib/dracut/dracut-initramfs-restore
 Type=oneshot
 RemainAfterExit=yes
