From 9d75fd3b62be29813c9c30a88e3f1bacc921847a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 18 Mar 2014 13:21:22 +0100
Subject: [PATCH] shutdown: if kexec failed, do a simple reboot

---
 modules.d/99shutdown/shutdown.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/99shutdown/shutdown.sh b/modules.d/99shutdown/shutdown.sh
index 162abcb9..1daf2ca3 100755
--- a/modules.d/99shutdown/shutdown.sh
+++ b/modules.d/99shutdown/shutdown.sh
@@ -113,6 +113,7 @@ case "$ACTION" in
     kexec)
         kexec -e
         warn "$ACTION failed!"
+        reboot -f -d -n
         ;;
     *)
         warn "Shutdown called with argument '$ACTION'. Rebooting!"
