From b9372137fd224ec57d44a4ba5cff0622e20fcab5 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 21 Jan 2015 13:39:28 +0100
Subject: [PATCH] network:ifup source team info if present

---
 modules.d/40network/ifup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/40network/ifup.sh b/modules.d/40network/ifup.sh
index 7836a6b3..524a3fb5 100755
--- a/modules.d/40network/ifup.sh
+++ b/modules.d/40network/ifup.sh
@@ -38,6 +38,7 @@ for i in /tmp/team.*.info; do
     [ -e "$i" ] || continue
     unset teamslaves
     unset teammaster
+    . "$i"
     for slave in $teamslaves ; do
         if [ "$netif" = "$slave" ] ; then
             netif=$teammaster
