From aea64fe178baa0035783e3e13b84dfc029958a89 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 1 Jul 2016 12:28:38 +0200
Subject: [PATCH] TEST-14-IMSM: add rd.device.timeout

---
 test/TEST-14-IMSM/test.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/TEST-14-IMSM/test.sh b/test/TEST-14-IMSM/test.sh
index a600e1c0..8e99b1f6 100755
--- a/test/TEST-14-IMSM/test.sh
+++ b/test/TEST-14-IMSM/test.sh
@@ -33,8 +33,8 @@ test_run() {
     client_run rd.auto rd.md.uuid=$MD_UUID rd.dm=0 || return 1
     # This test succeeds, because the mirror parts are found without
     # assembling the mirror itsself, which is what we want
-    client_run rd.auto rd.md.uuid=$MD_UUID rd.md=0 rd.md.imsm failme && return 1
-    client_run rd.auto rd.md.uuid=$MD_UUID rd.md=0 failme && return 1
+    client_run rd.auto rd.md.uuid=$MD_UUID rd.md=0 rd.md.imsm failme rd.device.timeout=40 && return 1
+    client_run rd.auto rd.md.uuid=$MD_UUID rd.md=0 failme rd.device.timeout=40 && return 1
     # the following test hangs on newer md
     client_run rd.auto rd.md.uuid=$MD_UUID rd.dm=0 rd.md.imsm rd.md.conf=0 || return 1
    return 0
