From 6dcb4867aeb463dd101947992b91781eec15e10f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 20 Oct 2016 15:27:53 +0200
Subject: [PATCH] README.md: update

---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 4c53f28d..9617eaf9 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-# dracut
+# dracut RHEL-7 branch
 
 dracut is an initramfs infrastructure.
 
