From fa861e2cbff8cde4fd55437f63f29d2b38c57d09 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 11 Jan 2017 13:47:24 +0100
Subject: [PATCH] dracut.conf.5.asc: add Warning about leading and trailing
 spaces in lists

---
 dracut.conf.5.asc | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/dracut.conf.5.asc b/dracut.conf.5.asc
index c20a341d..f4ad49d2 100644
--- a/dracut.conf.5.asc
+++ b/dracut.conf.5.asc
@@ -30,6 +30,11 @@ dracut command line options will overwrite any values set here.
 
 Configuration files must have the extension .conf; other extensions are ignored.
 
+[WARNING]
+====
+Space-separated lists have to have a leading and trailing space!
+====
+
 *dracutmodules+=*" __<dracut modules>__ "::
     Specify a space-separated list of dracut modules to call when building the
     initramfs. Modules are located in _/usr/lib/dracut/modules.d_.
