diff --git a/modules.d/95fcoe/fcoe-up.sh b/modules.d/95fcoe/fcoe-up.sh
--- a/modules.d/95fcoe/fcoe-up.sh	2019-08-05 09:17:59.369188642 +0100
+++ b/modules.d/95fcoe/fcoe-up.sh	2019-08-02 14:07:58.841210649 +0100
@@ -82,7 +82,8 @@
     sleep 1
 
     write_fcoemon_cfg
-    fcoemon --syslog
+    /sbin/initqueue --settled --unique --name start-fcoemon /sbin/start-fcoemon
+    /sbin/initqueue --timeout --unique --name start-fcoemon-timeout /sbin/start-fcoemon timeout
 elif [ "$netdriver" = "bnx2x" ]; then
     # If driver is bnx2x, do not use /sys/module/fcoe/parameters/create but fipvlan
     modprobe 8021q
@@ -93,7 +94,12 @@
 else
     vlan="no"
     write_fcoemon_cfg
-    fcoemon --syslog
+    /sbin/initqueue --settled --unique --name start-fcoemon /sbin/start-fcoemon
+    /sbin/initqueue --timeout --unique --name start-fcoemon-timeout /sbin/start-fcoemon timeout
 fi
 
+fcoe_mac=$(cat /sys/class/net/$netif/address)
+rm -f "/tmp/fcoe.wait.$netif"
+rm -f "/tmp/fcoe.wait.$fcoe_mac"
+
 need_shutdown
diff --git a/modules.d/95fcoe/module-setup.sh b/modules.d/95fcoe/module-setup.sh
--- a/modules.d/95fcoe/module-setup.sh	2019-08-05 09:17:59.389189756 +0100
+++ b/modules.d/95fcoe/module-setup.sh	2019-08-02 14:00:53.799636053 +0100
@@ -81,6 +81,7 @@
     inst "$moddir/fcoe-up.sh" "/sbin/fcoe-up"
     inst "$moddir/fcoe-edd.sh" "/sbin/fcoe-edd"
     inst "$moddir/fcoe-genrules.sh" "/sbin/fcoe-genrules.sh"
+    inst "$moddir/start-fcoemon.sh" "/sbin/start-fcoemon"
     inst_hook pre-trigger 03 "$moddir/lldpad.sh"
     inst_hook cmdline 99 "$moddir/parse-fcoe.sh"
     inst_hook cleanup 90 "$moddir/cleanup-fcoe.sh"
diff --git a/modules.d/95fcoe/parse-fcoe.sh b/modules.d/95fcoe/parse-fcoe.sh
--- a/modules.d/95fcoe/parse-fcoe.sh	2019-08-05 09:17:59.295184525 +0100
+++ b/modules.d/95fcoe/parse-fcoe.sh	2019-08-02 14:05:39.999578944 +0100
@@ -66,5 +66,11 @@
             warn "Invalid FCoE DCB option: $fcoe_dcb"
         fi
         . $(command -v fcoe-genrules.sh)
+
+        if [ -n "$fcoe_mac" ] ; then
+            > "/tmp/fcoe.wait.$fcoe_mac"
+        elif [ -n "$fcoe_interface" ]; then
+            > "/tmp/fcoe.wait.$fcoe_interface"
+        fi
     done
 fi
diff --git a/modules.d/95fcoe/start-fcoemon.sh b/modules.d/95fcoe/start-fcoemon.sh
new file mode 100755
--- /dev/null
+++ b/modules.d/95fcoe/start-fcoemon.sh	2019-08-02 14:42:17.967763238 +0100
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ -f /tmp/fcoe.started ]; then
+    exit 0
+fi
+
+# Start fcoemon if all FCoE interfaces have been configured
+# i.e. there are no more "wait" files.
+# Also start if we have hit a timeout.
+
+if [ ! "$1" = "timeout" -a stat /tmp/fcoe.wait.* &> /dev/null ]; then
+    exit 0
+fi
+
+fcoemon --syslog
+> /tmp/fcoe.started
